ui/src/components/Publication/SidebarCard/Actions/index.tsx

```diff
@@ -473,28 +473,32 @@ const Actions: React.FC<ActionProps> = (props): React.ReactElement => {
             </div>
 
             {user && user.email ? (
-                <>
-                    <Components.PublicationSidebarCardActionsButton
-                        label="Write review"
-                        onClick={() => {
-                            router.push({
-                                pathname: `${Config.urls.createPublication.path}`,
-                                query: {
-                                    for: props.publication.id,
-                                    type: 'PEER_REVIEW'
-                                }
-                            });
-                        }}
-                    />
-                    <Components.PublicationSidebarCardActionsButton
-                        label="Rate this publication"
-                        onClick={() => setShowRatingsModel(true)}
-                    />
-                    <Components.PublicationSidebarCardActionsButton
-                        label="Flag a concern with this publication"
-                        onClick={() => setShowRedFlagModel(true)}
-                    />
-                </>
+                props.publication.user.id !== user.id && (
+                    <>
+                        {props.publication.type !== 'PEER_REVIEW' && (
```

Comment:
If publication is of type peer review, then write a review option should not be shown. API will throw is attempting to link a peer review to another peer review.